Bookish.DataAccess/CreateAccountTable.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE dbo.Accounts (
+	Username varchar(20) PRIMARY KEY NOT NULL,
+	Pass varchar(20) NOT NULL
```

Comment:
I guess you could use the username as the id. But it's more common to have a separate user id 🙂 

Once you get to adding the website step the framework will handle the login for you. I wouldn't bother storing a password for now 🙂